digital-deck-ui/src/components/MainMenu.vue

```diff
@@ -181,30 +184,25 @@ export default {
       }
     },
 
-    async joinRoomForm() {
-      if (this.roomid!=null    ) {
-            this.$store.dispatch("joinSession", {
-            sessionId: this.roomid,
-            name: this.joinname,
-          })
-          .then(() => {
-             if (this.$store.getters.getGameInfo) {
-               console.log("You cant join");
-             }
-             else {
-             this.isLoaded = true;
-             }
-          });
-      }        
-      else {
-      console.log("Wrong room id enter again ");
+    joinRoomForm() {
+      if (this.roomid != null) {
+        this.$store.dispatch("joinSession", {
+          sessionId: this.roomid,
+          name: this.joinname,
+        }).then(() => {
+          if (!this.$store.getters.getJoinErrorMsg) {
+            this.isLoaded = true;
+          }
+        });
```

Comment:
I see that instead of specific blocks for each error you added a store variable, but they are not being used anywhere at the moment. Given that we don't have an error display on UI right now. It might be best idea to just console log for now.